test/WebJobs.Script.Tests.Integration/ScriptHostEndToEnd/NodeContentTests.cs

```diff
@@ -248,23 +248,23 @@ public NodeContentTests(TestFixture fixture)
         }
 
         [Fact]
-        public async Task ByteArrayXmlRaw()
+        public async Task ByteArrayXmlResponse()
         {
             var str = "asdf";
             var bytes = Encoding.UTF8.GetBytes(str);
             var base64 = Convert.ToBase64String(bytes);
-            var content = await Raw(bytes, "application/xml; charset=utf-8");
+            var content = await Response(bytes, "application/xml; charset=utf-8");
             Assert.Equal("asdf", content);
         }
 
         [Fact( Skip = "unsupported" )]
-        public async Task ObjectXmlResponse()
+        public async Task ObjectXmlResponse_Conneg()
```

Comment:
Few tests are being skipped. Can we enable them?

Reply:
unfortunately, no :( those tests have to do with different types of request objects instead of response body types, which wasn't in the scope of this PR